chia/cmds/wallet.py

```diff
@@ -941,15 +941,57 @@ def nft_transfer_cmd(
 )
 @click.option("-f", "--fingerprint", help="Set the fingerprint to specify which key to use", type=int)
 @click.option("-i", "--id", help="Id of the NFT wallet to use", type=int, required=True)
-def nft_list_cmd(wallet_rpc_port: Optional[int], fingerprint: int, id: int) -> None:
+@click.option(
+    "-o",
+    "--offset",
+    help="Skip NFTs from the beginning of the list",
+    type=int,
+    default=None,
```

Comment:
I wonder if this should be 0, but I guess we already decided `None`.  But maybe the data path could be reviewed and internally it could be made 0 asap?